cmd/bblfshctl/cmd/driver_remove.go

```diff
@@ -18,25 +18,46 @@ type DriverRemoveCommand struct {
 		Language string `positional-arg-name:"language" description:"language supported by the driver"`
 	} `positional-args:"yes"`
 
+	All bool `long:"all" description:"removes all the installed drivers"`
+
 	DriverCommand
 }
 
 func (c *DriverRemoveCommand) Execute(args []string) error {
+	ctx := context.Background()
 	if err := c.ControlCommand.Execute(nil); err != nil {
 		return err
 	}
 
-	r, err := c.srv.RemoveDriver(context.Background(), &protocol.RemoveDriverRequest{
-		Language: c.Args.Language,
-	})
-
-	if err == nil && len(r.Errors) == 0 {
-		return nil
+	langs := []string{c.Args.Language}
+	if c.All {
+		r, err := c.srv.DriverStates(ctx, &protocol.DriverStatesRequest{})
+		if err != nil {
+			return err
+		}
+
+		if err != nil || len(r.Errors) > 0 {
```

Comment:
Duplicated check of err (there is another above).

Reply:
good catch, fixed